net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/general.xml

```diff
@@ -5,4 +5,28 @@
         <type>checkbox</type>
         <help>This will activate the routing service.</help>
     </field>
+    <field>
+        <id>general.enablelogfile</id>
+        <label>create a logfile</label>
```

Comment:
@mimugmail this is probably something discussable. @fichtner what's your opinion here?